squape/squishserver.py

```diff
@@ -89,7 +90,17 @@ def _config_squishserver(self, config_option: str, params=None, cwd=None):
             f"Executing command: {' '.join(cmd)}",
             f"cwd: {cwd}",
         )
-        (exitcode, stdout, stderr) = self.remotesys.execute(cmd, cwd)
+
+        remote_os = self.remotesys.getOSName()
+        if remote_os == "Windows":
+            (exitcode, stdout, stderr) = self.remotesys.execute(cmd, cwd)
+        else:
+            squish_prefix = self.remotesys.getEnvironmentVariable("SQUISH_PREFIX")
```

Comment:
Is this necessary? I thin you could use `cwd` instead

Reply:
I used **self.location** instead of **squish_prefix**, since the location can be passed to the constructor as a parameter.